config/seed-data/loadroles.js

```diff
@@ -47,7 +47,7 @@ module.exports.sysroles = function () {
 var goodRoleCode2 = function (code) {
 	// leave these codes...
 	// and any project roles with these codes.
-	var goodRoles = ['mem', 'admin', 'public', 'user', 'eao', 'proponent', 'invitee'];
+	var goodRoles = ['mem', 'admin', 'public', 'user', 'proponent', 'invitee'];
```

Comment:
Do we need to remove role _mem_ from here? And is it right to delete role _eao_?